cellml pip does not install on linux any more

Hi all,
unfortunately you seem to have broken the pypi package for linux.

I.e.
```
pip install libcellml
```
only installs a very old version.
```
Successfully installed libcellml-0.2.0.dev21
```
but not the latest version on python 3.9 on linux (ubuntu), i.e., https://pypi.org/project/libcellml/

Installation on python 3.10 does not work at all in linux (ubuntu).

Please fix this issues, because the library is not usable at the moment on linux without the pypi package.

Best Matthias

Sounds good. Seems just to be an issue with the pypi upload as you can see from the failing CI:
https://github.com/cellml/libcellml/actions/runs/2907912016
```
25hWARNING  Error during upload. Retry with the --verbose option for more details. 
ERROR    HTTPError: 400 Bad Request from https://upload.pypi.org/legacy/        
         Binary wheel 'libcellml-0.3.100-cp37-cp37m-linux_x86_64.whl' has an    
         unsupported platform tag 'linux_x86_64'.                               
Error: Process completed with exit code 1.
```

As I understand things you have to use manylinux as stated here: https://stackoverflow.com/questions/59451069/binary-wheel-cant-be-uploaded-on-pypi-using-twine
> You cannot upload a linux_*.whl wheel to PyPI (reasons [here](https://www.python.org/dev/peps/pep-0513/#rationale)).
>You'll need to convert this platform-specific wheel into a manylinux wheel via the [manylinux project](https://github.com/pypa/manylinux) and the [auditwheel tool](https://github.com/pypa/auditwheel).

Best Matthias

--- a/.github/workflows/deploy-on-release.yml
+++ b/.github/workflows/deploy-on-release.yml
@@ -342,7 +342,7 @@ jobs:
             MACOSX_DEPLOYMENT_TARGET=11.0
             LIBCELLML_VERSION_TAG=${{ steps.setup.outputs.tag }}
           CIBW_REPAIR_WHEEL_COMMAND_MACOS: ""
-          CIBW_REPAIR_WHEEL_COMMAND_LINUX: ""
+          # CIBW_REPAIR_WHEEL_COMMAND_LINUX: ""
 
       - name: Upload Python wheels as artifacts
         if: needs.setup-jobs.outputs.binaries-destination == 'Artifact'

--- a/.github/workflows/deploy-on-release.yml
+++ b/.github/workflows/deploy-on-release.yml
@@ -342,7 +342,8 @@ jobs:
             MACOSX_DEPLOYMENT_TARGET=11.0
             LIBCELLML_VERSION_TAG=${{ steps.setup.outputs.tag }}
           CIBW_REPAIR_WHEEL_COMMAND_MACOS: ""
-          # CIBW_REPAIR_WHEEL_COMMAND_LINUX: ""
+          CIBW_BEFORE_BUILD_LINUX: pip install renamewheel
+          CIBW_REPAIR_WHEEL_COMMAND_LINUX: renamewheel -w {dest_dir} {wheel}
 
       - name: Upload Python wheels as artifacts
         if: needs.setup-jobs.outputs.binaries-destination == 'Artifact'

--- a/docs/changelogs/changelog_v0.3.100.rst
+++ b/docs/changelogs/changelog_v0.3.100.rst
@@ -1,5 +1,5 @@
-libCellML latest Changelog
-==========================
+libCellML v0.3.100 Changelog
+============================
 
 Infrastructure
 --------------

--- a/docs/changelogs/changelog_v0.3.99.rst
+++ b/docs/changelogs/changelog_v0.3.99.rst
@@ -1,5 +1,5 @@
-libCellML latest Changelog
-==========================
+libCellML v0.3.99 Changelog
+===========================
 
 Bug
 ---

